Resources/NetHookAnalyzer2/NetHookAnalyzer2/NetHookItem.cs

```diff
@@ -53,6 +54,8 @@ public bool LoadFromFile(FileInfo fileInfo)
 				return false;
 			}
 
+			Timestamp = File.GetLastWriteTime(fileInfo.FullName);
```

Comment:
The `fileInfo` object has a `LastWriteTime` property. Using this would be more optimal as .NET doesn't have to revalidate the path.